devicetypes/smartthings/smartsense-motion-temp-sensor.src/smartsense-motion-temp-sensor.groovy

```diff
@@ -194,14 +194,9 @@ private Map getBatteryResult(rawValue) {
 	]
 
 	def volts = rawValue / 10
-	def descriptionText
 
-    if (rawValue == 0 || rawValue == 255) {}
-	else {
-		if (volts > 3.5) {
-			result.descriptionText = "${linkText} battery has too much power (${volts} volts)."
-		}
-		else if (volts > 0){
+	if (!(rawValue == 0 || rawValue == 255)) {
+		if (volts > 0) {
```

Comment:
`if (volts > 0)` isn't needed since we already check to make sure rawValue is not 0 and it won't be negative.